### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by akuinu**
Added reminder about- web instead of worker dyno.
Common mistake by some people that's answer is hidden away in issues.

### CODE REVIEW

**Comment by synicalsyntax** [comment]
Hi, thanks for the PR! @akuinu 

The notice reads fine at a glance, but I'm not sure if this is the right place to put this notice in the guide. If you read it as a whole, you'll notice that at this point we've only just created the Procfile and haven't actually created the Heroku application nor remotely built or launched the application yet. In order to flow with the guide more, you'd probably have to relocate it to a later section.

**Comment by akuinu** [comment]
Relocated the information to Troubleshooting section near the end of the file.
Also added some basic troubleshooting hints

File: README.md [hunk 1]
```diff
@@ -489,6 +489,31 @@ $ git push heroku master
...
+
+```sh
+$ heroku ps:scale web=0
+Scaling dynos... done, now running web at 0:Free
+$ heroku ps:scale worker=1
+Scaling dynos... done, now running worker at 1:Free
+```
+
+If that is not your problem then you need to do some debugging on your own - 
+it is impossible to predict all potential problems.
```
Comment (synicalsyntax) [inline README.md:516]: Insert a new line afterwards.

File: README.md [hunk 2]
```diff
@@ -489,6 +489,31 @@ $ git push heroku master
...
+`Error R10 (Boot timeout) -> Web process failed to bind to $PORT within 60 seconds of launch`
+
+Then you should check if you are running your app correct dyno - It should running it on worker and not on web, 
+you can read about different types of dyons [here](https://devcenter.heroku.com/articles/dynos).
+To find out what kind on dyno your application is running you use can go to 
+[Heroku](https://dashboard.heroku.com/apps)
+website and check under your app Resources tab or using command line `heroku ps`. 
+There should be only one worker dyno - if not you can change it using UI or by following commands: 
+
+```sh
```
Comment (synicalsyntax) [inline README.md:508]: Can you be more explicit on what each of the commands does? Also, I believe you can combine scaling web and worker commands together; there's no need for 2 separate commands.

File: README.md [hunk 3]
```diff
@@ -489,6 +489,31 @@ $ git push heroku master
...
 up, deploying to, and hosting your bot on Heroku!
 
+## Troubleshooting
+
+Sometimes mistakes happen and like with most programming related problem - the solution is reading logs and documentation. 
+To check your Heroku app logs you can use command `heroku logs --tail` and read what might be cause of your problem.
+
+In case you see following line in your logs:
+
+`Error R10 (Boot timeout) -> Web process failed to bind to $PORT within 60 seconds of launch`
```
Comment (synicalsyntax) [inline README.md:499]: Can you make this a full codeblock?

File: README.md [hunk 4]
```diff
@@ -489,6 +489,31 @@ $ git push heroku master
...
+
+Then you should check if you are running your app correct dyno - It should running it on worker and not on web,
```
Comment (synicalsyntax) [inline README.md:501]: See my previous comment about rephrasing; the dash isn't correct punctuation, "It should running" isn't proper grammar, "dyons" is misspelled, etc.

File: README.md [hunk 5]
```diff
@@ -489,6 +489,31 @@ $ git push heroku master
...
+There should be only one worker dyno - if not you can change it using UI or by following commands: 
+
+```sh
+$ heroku ps:scale web=0
+Scaling dynos... done, now running web at 0:Free
+$ heroku ps:scale worker=1
+Scaling dynos... done, now running worker at 1:Free
+```
+
+If that is not your problem then you need to do some debugging on your own -
```
Comment (synicalsyntax) [inline README.md:515]: This phrase isn't very encouraging and sounds slightly condescending. Can you try rephrasing in a more positive manner?

File: README.md [hunk 6]
```diff
@@ -489,6 +489,31 @@ $ git push heroku master
...
+
+In case you see following line in your logs:
+
+`Error R10 (Boot timeout) -> Web process failed to bind to $PORT within 60 seconds of launch`
+
+Then you should check if you are running your app correct dyno - It should running it on worker and not on web, 
+you can read about different types of dyons [here](https://devcenter.heroku.com/articles/dynos).
+To find out what kind on dyno your application is running you use can go to 
+[Heroku](https://dashboard.heroku.com/apps)
+website and check under your app Resources tab or using command line `heroku ps`.
```
Comment (synicalsyntax) [inline README.md:505]: The guide focuses on the CLI instead of the web UI, so you can remove any references to the web app.

File: README.md [hunk 7]
```diff
@@ -489,6 +489,31 @@ $ git push heroku master
 If the app is deployed successfully, congratulations! You've finished setting
 up, deploying to, and hosting your bot on Heroku!
 
+## Troubleshooting
+
+Sometimes mistakes happen and like with most programming related problem - the solution is reading logs and documentation.
```
Comment (synicalsyntax) [inline README.md:494]: The grammar and punctuation here in this paragraph isn't correct, so it makes it awkward to read; for example, the dash would be better suited as a comma. Can you try rephrasing?

**Review by synicalsyntax — CHANGES_REQUESTED** [review]
Hi, thanks for updating! I commented with some areas that need improvement. 

Also, before I merge, can you please amend your original commit message to "Add Troubleshooting section to address web dyno issues." or something similar? Additionally, I'd like to request that you squash the changes in your second commit into your first commit so that there isn't 2 commits.

Thanks for your work so far!

**Comment by synicalsyntax** [comment]
Closing PR due to inactivity; if you want to work on it further, feel free to amend your commits and reopen. Thanks!

**Pull request closed without merging**